src/usb_iss/usb_iss.py

```diff
@@ -49,7 +49,14 @@ def __init__(self, dummy=False):
         self.io = IO(self._drv)
         self.spi = SPI(self._drv)
         self.serial = Serial(self._drv)
-
+    
+    def get_port(self, vid=defs.VID_USB_ISS, pid=defs.PID_USB_ISS):

```

Comment:
I think this belongs in driver.py, with the other pyserial-specific code.

Reply:
Is there any use case for allowing a VID/PID to be passed in? Could just remove these parameters and always use the USB_ISS constants.